GitSavvy.sublime-settings

```diff
@@ -125,5 +125,12 @@
 
         NOTE: Changes you make may break in-view functionality, so use with caution.
      */
-    "git_graph_args": ["log", "--oneline", "--graph", "--all", "--decorate"]
+    "git_graph_args": ["log", "--oneline", "--graph", "--all", "--decorate"],
+
+    /*
+        Change this to `true` to pick the output type (e.g. "Show commit" or "Show diff") when 
+        selecting a commit from `git: log`.
+    */
+    "show_log_output_options": false
```

Comment:
@divmain I set this to false to preserve the current behaviour, but if you would prefer to make the new behaviour the default I can change this to `true`.